flowzone.yml

```diff
@@ -2135,14 +2135,15 @@ jobs:
         github_publish,
         project_types,
         versioned_source,
-        website_build
+        website_build,
       ]
     # avoid being skipped when dependencies are skipped
     # this job must always run so branch protection rules can depend on it
     if: |
       always() &&
       needs.event_types.outputs.do_draft == 'true' &&
-      inputs.protect_branch == true
+      inputs.protect_branch == true &&
+      github.event.pull_request.draft == false
```

Comment:
> this job must always run so branch protection rules can depend on it

this comment is violated since the conditions require that the PR not be a draft. Maybe change line 2141 to be more clear.